static/docs/xiangqi.md

```diff
@@ -143,21 +143,21 @@ Sometimes, some players would use a "." instead of "=".
 
 ### Disambiguation
 
-If two of the same piece are in the same file, then a + or - to specify if it's the more advanced piece (+) or the piece that's at a lower rank (-). For example, if two chariots are on the same file but on opposite sides of the river, and you wanted to move to the farther one to the center, that would be R+=5.
+If two of the same pieces are in the same file, then a + or - to specify if it's the more advanced piece (+) or the piece that's at a lower rank (-). For example, if two chariots are on the same file but on opposite sides of the river, and you wanted to move to the farther one to the center, that would be R+=5.
```

Comment:
I don't think simply changing it to "pieces" is correct here. This sentence is about having two pieces of the same type **on** (yeah this should be changed too) the same file.

Reply:
Should be "If two pieces of the same type ..."